plugins/vpp/vppcallmock/vpp_ctx_mock.go

```diff
@@ -165,9 +165,7 @@ func (ctx *TestCtx) MockReplies(dataList []*HandleReplies) {
 				sendControlPing = dataMock.Ping
 				if len(dataMock.Messages) > 0 {
 					log.DefaultLogger().Infof(" MOCK HANDLER: mocking %d messages", len(dataMock.Messages))
-					for _, msg := range dataMock.Messages {
-						ctx.MockVpp.MockReply(msg)
-					}
+					ctx.MockVpp.MockReply(dataMock.Messages...)
```

Comment:
How come it was working until now when the `MockReply` creates additional reply for each time it's called? Or was it unused and was just unnoticed?

Reply:
yeah, unused until now